content/tutorials/build-apps/build-a-desktop-wallet-in-js.md

```diff
@@ -0,0 +1,211 @@
+
+5. Create another file in helpers folder named get-ledger-details.js which will be used to fetch the ledger details and copy the code written below to the file.
+
+{{ include_code("_code-samples/build-a-wallet/js/src/helpers/get-ledger-details.js", language="js") }}
+
+In the function `getLedgerDetails`, we are using the [ledger](https://xrpl.org/ledger.html) that will fetch the details of the latest ledger, you can tweak the code to fetch the details of a specific ledger and also to fetch account details and other information.
+
+
+6. Now, let's add the code to index.js file to fetch the account and ledger details and display them on the home page. Copy the code written below to the index.js file.
+
+{{ include_code("_code-samples/build-a-wallet/js/index.js", language="js") }}
+
+In the code above, we imported the `getWalletDetails` and `getLedgerDetails` functions from the get-wallet-details.js and get-ledger-details.js files respectively. Added event listners to the buttons for redirection. Used the `setInterval` method to call the getLedgerDetails function every 4 seconds. (Note: The ledger's estimate close time is 4 seconds, so we are calling the function every 4 seconds to get the latest ledger details. This can be adjusted according to your needs.)
+
+Up next we need to update the HTML and CSS for the home page to give it the XRPL-look and feel. 
+
+7. Copy the contents of the [index.html](_code-samples/build-a-wallet/js/index.html) file to your index.html file.
+
+For this tutorial, we're using basic HTML and pre-loaded CSS files. If you'd like to change the aesthetic, you can replace it with any CSS framework of your choice!
+
+8. Copy the code from the [index.css](_code-samples/build-a-wallet/js/index.css) file to your project's index.css file to use the pre-made stylings.
+
+### Step 3: Creating the Send XRP Page
+
+Till now, we have created the home page that displays the account and ledger details. In this step, we will create the Send XRP page that will allow users to send XRP to another account.
+
+![Send XRP Page Screenshot](img/js-wallet-send-xrp.png)
+
+1. Create a folder named send-xrp in the src directory.
+
+2. Add two files named send-xrp.js and send-xrp.html. You can also create a separate css file depending on your choice. 
+
+3. Copy the contents of the [send-xrp.html](_code-samples/build-a-wallet/js/src/send-xrp/send-xrp.html) file to your send-xrp.html file. The provided HTML code includes three input fields for the destination address, amount, and destination tag, each with their corresponding labels.
+
+4. Now that we have the HTML code, let's add the JavaScript code. In helpers folder, add a new file named submit-transaction.js and copy the code written below to the file.
```

Comment:
Similar to the previous suggestion, it's helpful to describe what the code you're copying is going to accomplish before you ask them to copy it. 

Also, changing "add" to "create" if we're making a new file because "add" could mean to copy an existing file. 

```suggestion
4. Now that we have the HTML code, let's add the JavaScript code. In the `helpers` folder, create a new file named `submit-transaction.js` and copy the code written below to the file.
```